Mathlib/Algebra/Group/Hom/Instances.lean

```diff
@@ -18,8 +18,6 @@ is a commutative group. We also prove the same instances for additive situations
 
 Since these structures permit morphisms of morphisms, we also provide some composition-like
 operations.
-
-Finally, we provide the `Ring` structure on `AddMonoid.End`.
```

Comment:
I guess this and the comment change below could be split to their own PR and merged immediately, since these changes reflect what is already the status quo

Reply:
@Ruben-VandeVelde do you mind doing this?